packages/types/lib/responses.ts

```diff
@@ -18,13 +19,19 @@ import {
   Video
 } from "@lib/types";
 
-// Response that should always return an error, used by westegg
+//
+// Generic Responses
+// --------------------------------------------------
 export type ErrorResponse = {
   error: WestEggError;
 };
 
 export type Response<T = {}> = T | ErrorResponse;
 
+export type CursorResponse<T = {}> =
+  | { content: T; cursor: ObjectID; limit: number }
```

Comment:
So, in the case where there are no more values in the collection, `cursor` should be set to `undefined`. As such, we should actually make `cursor` optional here.